src/servers/ZoneServer2016/managers/constructionmanager.ts

```diff
@@ -347,13 +346,14 @@ export class ConstructionManager {
     return false;
   }
   detectPOIPlacement(
+    server: ZoneServer2016,
     itemDefinitionId: number,
     position: Float32Array,
     client: Client,
     isInsidePermissionedFoundation: boolean
   ): boolean {
     if (client.isDebugMode) return false;
-    if (this.allowPOIPlacement) return false;
+    if (server.isNoBuildInPois) return false;
```

Comment:
wouldbe better to do this directly when detectPOIPlacement method is called

Reply:
Done 👍